Jointify/Views/Screens/ProcessingView.swift

```diff
@@ -116,17 +118,31 @@ struct ProcessingView: View {
                 print("Analysing frame \(frameCount)/\(frames.count)")
                 
                 let drawnImage = poseNet.predict(frame)
+                
+                let outputQualityAcceptable = poseNet.assessOutputQuality()
 
-                returnMeasurementFrames.append(
-                    MeasurementFrame(
-                        degree: poseNet.calcAngleBetweenJoints(),
-                        image: drawnImage
+                // Only append measurement frame if it fulfills quality criteria
+                if outputQualityAcceptable {
+                    self.removedFramesCounter += 1
+                    returnMeasurementFrames.append(
+                        MeasurementFrame(
+                            degree: poseNet.calcAngleBetweenJoints(),
+                            image: drawnImage
+                        )
                     )
-                )
+                }
+                
                 self.progress += 1
 
             }
             
+            let acceptedFramesPercentage = Double(self.removedFramesCounter) / Double(self.progress)
+            
+            if acceptedFramesPercentage < self.acceptedFramesThreshold {
+                // TODO: throw proper error message at user
```

Comment:
No TODO to be pushed into `develop`. Add an issue if not yet done and remove line
```suggestion
```